hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcherTest.java

```diff
@@ -410,8 +451,7 @@ void testDataNotSetFails() {
         final var context = new PreHandleContext(accountStore, txBody, payer);
 
         // then
-        assertThatThrownBy(() -> dispatcher.dispatchPreHandle(tracker, context))
-                .isInstanceOf(UnsupportedOperationException.class);
+        assertThrows(UnsupportedOperationException.class, () -> dispatcher.dispatchPreHandle(tracker, context));
```

Comment:
We decided to use AssertJ. I understand that some code still uses other assertions and there are more important things that cleaning this up, but we should not replace AssertJ assertions with other ones.

Reply:
Using AssertJ libraries was causing errors here for some reason, but I'll go back and double check

Reply:
Nope, actually I think I just used the wrong library here. Changed to AssertJ. Thanks!